docs/hyperopt.md

```diff
@@ -376,6 +381,41 @@ minimal_roi = {
     }
 ```
 
+If you are optimizing ROI, Freqtrade creates the 'roi' optimization hyperspace for you -- it's the hyperspace of components for the ROI tables. By default, each ROI table generated by the Freqtrade consists of 4 rows (steps) with the values that can vary in the following ranges:
+
+| # | minutes | ROI percentage |
+|---|---|---|
+| 1 | always 0 | 0.03...0.31 |
+| 2 | 10...40 | 0.02...0.11 |
+| 3 | 20...100 | 0.01...0.04 |
+| 4 | 30...220 | always 0 |
+
+This structure of the ROI table is sufficient in most cases. Override the `roi_space()` method defining the ranges desired if you need components of the ROI tables to vary in other ranges.
+
+Override the `generate_roi_table()` and `roi_space()` methods and implement your own custom approach for generation of the ROI tables during hyperoptimization in these methods if you need a different structure of the ROI table or other amount of rows (steps) in the ROI tables.
+
+### Understand Hyperopt Stoploss results
+
+If you are optimizing stoploss values, your result will look as follows and include stoploss:
```

Comment:
How are you optimizing stoploss values? (same for ROI above)?

maybe something like

```
If you are optimizing stoploss values (search-space includes `all` or `stoploss`), your result will look as follows and include stoploss:
```
?

Reply:
Agree, this adds some more clearness, done (also for the 'roi' section).